src/bda/plone/orders/mailnotify.py

```diff
@@ -432,6 +443,37 @@ def notify_booking_cancelled(event, who=None):
             'kw "who" mus be one out of ("customer", "shopmanager")'
         )
 
+class StockThresholdReachedCB(object):
+
+    def __init__(self, event):
+        self.event = event
+
+    def __call__(self, *args):
+        items_stock_threshold_reached_text = ""
+        items = self.event.items_stock_threshold_reached
+        for item_attrs in items:
+            title = item_attrs['title']
+            remaining_stock = item_attrs['remaining_stock_available']
+            items_stock_threshold_reached_text += "%s (Remaining stock: %s)\n" %(title, remaining_stock)
+
+        return items_stock_threshold_reached_text
+
+
+def notify_stock_threshold_reached(event, who=None):
+    """Send notification mail when item is getting out of stock.
+    """
+    order_data = OrderData(event.context, uid=get_order_uid(event))
+    templates = dict()
+    templates.update(get_stock_threshold_reached_templates(event.context))
+    templates['items_stock_threshold_reached_text_cb'] = StockThresholdReachedCB(event)
+
+    if who == 'shopmanager':
+        do_notify_shopmanager(event.context, order_data, templates)
+    else:
+        raise ValueError(
+            'kw "who" mus be one out of ("customer", "shopmanager")'
```

Comment:
There is no either customer or shopmanager if we send only if who is shopmanager, right?